src/panels/config/integrations/integration-panels/zwave_js/dialog-zwave_js-add-node.ts

```diff
@@ -211,16 +283,40 @@ class DialogZWaveJSAddNode extends LitElement {
       this._status = "failed";
     }
     if (message.event === "inclusion stopped") {
-      if (this._status !== "finished") {
-        this._status = "";
-      }
-      this._unsubscribe();
+      // we get the inclusion stopped event before the node added event
+      // during a successful inclusion. so we set a timer to wait 3 seconds
+      // to give the node added event time to come in before assuming it
+      // timed out or was cancelled and unsubscribing.
+      this._stoppedTimeout = setTimeout(() => {
+        if (!this._nodeAdded) {
+          this._status = "";
```

Comment:
Should be a fairly rare occurrence (either it timed out because you didn't push the button on the node to pair it, or you cancelled the inclusion from a different tool like zwavejs2mqtt)